### ISSUE TITLE
binstar-token invalid

### ISSUE THREAD

**Issue by calum-chamberlain** [conda-forge/eqcorrscan-feedstock#2]
Currently circle-ci builds are failing with the following message:
```
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 133, in add_release
    aserver_api.release(username, package_name, version)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/__init__.py", line 377, in release
    self._check_response(res)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/__init__.py", line 216, in _check_response
    raise ErrCls(msg, res.status_code)
binstar_client.errors.NotFound: ("release version=u'0.2.5' does not exist", 404)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/bin/anaconda", line 6, in <module>
    sys.exit(binstar_client.scripts.cli.main())
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/scripts/cli.py", line 88, in main
    description=__doc__, version=version)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/scripts/cli.py", line 66, in binstar_main
    return args.main(args)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 242, in main
    args=args)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 180, in upload_package
    add_release(aserver_api, args, username, package_name, version, release_attrs)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 138, in add_release
    create_release(aserver_api, username, package_name, version, release_attrs)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 42, in create_release
    aserver_api.add_release(username, package_name, version, [], announce, release_attrs)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/__init__.py", line 417, in add_release
    self._check_response(res)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/__init__.py", line 216, in _check_response
    raise ErrCls(msg, res.status_code)
binstar_client.errors.Unauthorized: ('Authorization token is no longer valid', 401)
Traceback (most recent call last):
  File "/opt/conda/bin/upload_or_check_non_existence", line 122, in <module>
    main()
  File "/opt/conda/bin/upload_or_check_non_existence", line 111, in main
    upload(cli, meta, owner, channel)
  File "/opt/conda/bin/upload_or_check_non_existence", line 62, in upload
    env=os.environ)
  File "/opt/conda/lib/python3.6/subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['anaconda', '--quiet', '-t', 'binstar.token', 'upload', '/feedstock_root/build_artefacts/linux-64/eqcorrscan-0.2.5-np111py27_0.tar.bz2', '--user=conda-forge', '--channel=main']' returned non-zero exit status 1.

./ci_support/run_docker_build.sh returned exit code 1
```
This is the same for travis (and likely for appveyor).

I think this is related to this [issue](https://github.com/conda-forge/conda-forge.github.io/issues/426).

This release is not totally urgent as we have released via pypi, but it would be good to get it updated @conda-forge/core

**Comment by jakirkham** [comment]
The token has been revoked. Please see this [comment]( https://github.com/conda-forge/conda-forge.github.io/issues/426#issuecomment-321897876 ) for details.

**Comment by djsutherland** [comment]
The tokens got [automatically updated](https://github.com/conda-forge/eqcorrscan-feedstock/commit/b03bbca94d03919ad310de03362b9deaee0f940e); if you want to release 0.2.5, you can merge a PR with an empty commit `git commit --allow-empty -m "empty commit to force build"`, or you can go ahead and just release 0.2.6.

**Comment by calum-chamberlain** [comment]
Released 0.2.6. Thanks guys!
